conduct/reporting_events.md

```diff
@@ -0,0 +1,49 @@
+# Project Jupyter Code of Conduct - Reporting Guide - Jupyter Events
+
+You can make a personal report by:
+
+**Phone**: Provided at beginning of your event. This phone number will be
+monitored for the duration of the event.
+
+**In Person**: Contact a staff member, identified by STAFF badges.
+
+When taking a personal report, our staff will ensure you are safe and cannot be
+overheard. They may involve other event staff to ensure your report is managed
+properly. This can be uncomfortable, but we'll handle it respectfully and you
+can bring someone to support you.
+
+**Email**: Send a message to
+[*conduct@jupyter.org*](mailto:conduct@jupyter.org) and/or local organizer’s
+email (best to do both to ensure both prompt local response and that the
+regular committee is immediately in the loop).
+
+Our team will help you contact hotel/venue security, local law enforcement,
+local support services, provide escorts, or otherwise assist you to feel safe
```

Comment:
> ...local law enforcement, local support (which could include services and transportation as available), or otherwise respectfully assist you to feel safe...

I totally agree that we should help in ways we can. I think your wording, which mentions as an option but does not unconditionally commit us, is a sufficient balance between practicality and help.